### ISSUE TITLE
Can't use "graphical" tools like top / htop

### ISSUE THREAD

**Issue by davibe** [Shougo/vimshell.vim#186]
I am not sure what's wrong with those but i see just one line and everything else is black. See here

![screenshot 2015-02-04 10 10 14](https://cloud.githubusercontent.com/assets/123929/6037610/08563c22-ac56-11e4-8b91-117ddd2ed182.PNG)

**Comment by Shougo** [comment]
Yes.  It is well known problem.  But it is hard to fix.  I don't have much time for it.

**Comment by Shougo** [comment]
You should use `texe top`.  But it is not perfect.
You should not use graphical tools in vimshell.

**Comment by davibe** [comment]
Oki, thank you.
